flask_mongorest/views.py

```diff
@@ -20,15 +20,23 @@
 except NameError:
     text_type = str # Python 3
 
+def get_exception_message(e):
```

Comment:
The "what" of this function is clear, but the "why" isn't – maybe it's worth adding a comment/docstring explaining why we need this?